### PR TITLE
Point to correct configuration file for one-click webapp

### PR DESCRIPTION
**Pull request by whikloj**
**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2422

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
Points to correct configuration file to allow for one-click property to filter into the packaged webapp.

# How should this be tested?

1. Build everything : `mvn clean install`
1. Build fcrepo-webapp : `mvn clean install -Pone-click`
1. Double click the file `target/fcrepo-4.8.0-SNAPSHOT-jetty-console.jar` or `java -jar target/fcrepo-4.8.0-SNAPSHOT-jetty-console.jar`
1. Press the Start button

Did the repository start? Then 👍 

Example:
* Does this change require documentation to be updated? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code?no

# Interested parties
@awoods

### CODE REVIEW

**Comment by birkland** [comment]
WOW, what a difference the PR format makes in practice, nice @whikloj .  I have no time at the moment, but this just made me want to dive in anyway.  

I actually get an error consistently, seemingly unrelated to anything in this PR, but let's see what Travis says:

<pre>
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Fedora Commons 4 :: Parent POM ..................... SUCCESS [  0.188 s]
[INFO] Fedora Commons 4 ................................... SUCCESS [  2.908 s]
[INFO] Fedora Repository BOM .............................. SUCCESS [  0.308 s]
[INFO] Fedora Repository JCR BOM .......................... SUCCESS [  0.297 s]
[INFO] Fedora Repository BOMs ............................. SUCCESS [  0.280 s]
[INFO] Fedora Repository Kernel API ....................... FAILURE [  3.105 s]
[INFO] Fedora Repository Metrics Module ................... SKIPPED
[INFO] Fedora Repository Configurations Module ............ SKIPPED
[INFO] Fedora Repository Kernel Implementation (ModeShape)  SKIPPED
[INFO] Fedora Event Serialization ......................... SKIPPED
[INFO] Fedora Repository JMS Module ....................... SKIPPED
[INFO] Fedora Repository HTTP Commons Module .............. SKIPPED
[INFO] Fedora Repository Authorization Commons Module ..... SKIPPED
[INFO] Fedora Repository HTTP API Module .................. SKIPPED
[INFO] Fedora Repository Deployable Web Application ....... SKIPPED
[INFO] Fedora Repository LDP integration tests Module ..... SKIPPED
[INFO] Fedora Repository RDF integration tests Module ..... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.999 s
[INFO] Finished at: 2017-04-13T17:25:38-04:00
[INFO] Final Memory: 49M/523M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.18.1:test (default-test) on project fcrepo-kernel-api: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.18.1:test failed: There was an error in the forked process
[ERROR] java.lang.NoClassDefFoundError: org/apache/jena/graph/Node
[ERROR] at java.lang.Class.getDeclaredMethods0(Native Method)
[ERROR] at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
[ERROR] at java.lang.Class.getDeclaredMethods(Class.java:1975)
[ERROR] at org.apache.maven.surefire.common.junit4.JUnit4TestChecker.checkforTestAnnotatedMethod(JUnit4TestChecker.java:91)
[ERROR] at org.apache.maven.surefire.common.junit4.JUnit4TestChecker.lookForTestAnnotatedMethods(JUnit4TestChecker.java:80)
[ERROR] at org.apache.maven.surefire.common.junit4.JUnit4TestChecker.isValidJUnit4Test(JUnit4TestChecker.java:72)
[ERROR] at org.apache.maven.surefire.common.junit4.JUnit4TestChecker.accept(JUnit4TestChecker.java:52)
[ERROR] at org.apache.maven.surefire.util.DefaultScanResult.applyFilter(DefaultScanResult.java:97)
[ERROR] at org.apache.maven.surefire.junit4.JUnit4Provider.scanClassPath(JUnit4Provider.java:222)
[ERROR] at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:107)
[ERROR] at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
[ERROR] at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
[ERROR] at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
[ERROR] Caused by: java.lang.ClassNotFoundException: org.apache.jena.graph.Node
[ERROR] at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
[ERROR] at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[ERROR] at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
[ERROR] at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
</pre>

**Comment by whikloj** [comment]
@birkland strange, I am not getting the same issue. I'll try a fresh clone just in case something has gotten funky.

**Comment by birkland** [comment]
Ok, I'm going to have to dig in.  I blew everything away and even deleted the local maven repo - it does not make sense; but I'd be shocked if it has anything to do with this PR. 

I am building on a Windows machine, and am tired enough that I could have done something entirely stupid as well.  I *did* confirm that I'm building against this PR, though :)

**Comment by birkland** [comment]
OK.  It built successfully on a new git clone I made elsewhere.  Then I went back to the original directory _without changing anything_ and it built fine there too.  

The one-click jar works, so 👍 

I'm just going to back away slowly, and enjoy a beverage from my trusty, reliable, 77 year old refrigerator.

File: fcrepo-webapp/pom.xml [hunk 1]
```diff
@@ -500,7 +500,7 @@
             <directory>src/main/resources</directory>
             <filtering>true</filtering>
             <includes>
-              <include>**/repo.xml</include>
+              <include>**/fcrepo-config.xml</include>
```
Comment (awoods) [inline fcrepo-webapp/pom.xml]: I am glad to see this was easier than creating a jetty-console-maven-plugin extension :)

Comment (whikloj) [inline fcrepo-webapp/pom.xml]: Yep, sometimes I forget computers are not as smart as I think they are. 😄

**Pull request merged**